src/task.c

```diff
@@ -15,6 +15,7 @@
     along with Restraint.  If not, see <http://www.gnu.org/licenses/>.
 */
 
+#define _XOPEN_SOURCE 500
```

Comment:
Question only:
Which symbol are we missing here that we need `_XOPEN_SOURCE 500`?

Reply:
@danrodrig responded `localtime_r`. Thanks